src/net/constants.rs

```diff
@@ -9,4 +11,4 @@ pub const FRAGMENT_SIZE_DEFAULT: u16 = 1024;
 ///
 /// This is not strictly guaranteed -- there may be less room in an ethernet frame than this due to
 /// variability in ipv6 header size.
```

Comment:
ipv6 because we should work from the biggest values. ipv4 is just 20 bytes so we are safer when we have  an MTU with more space by calculating with the size of IPV6 wich is 40 bytes.